CONTRIBUTING.rst

```diff
@@ -12,16 +12,35 @@ log and by doing so you agree to the above.
 
 Getting access to the main project
 ----------------------------------
+
 Direct commit access to the project is currently restricted to core developers.
-All other contributions should be done through
-`Merge Requests <https://docs.gitlab.com/ee/user/project/merge_requests/getting_started.html>`_.
+All other contributions should be done through pull requests using the standard github mechanisms.
 
 Creating a Local Environment for Development
 --------------------------------------------
 
-[To be completed]
+This project requires the `mantid-framework <https://anaconda.org/mantid/mantid-framework>`_ conda package, as well as python3.6+ and qt5.
+These can be installed using the commands
+
+.. code-block:: bash
+
+   conda create -n refred -c mantid/label/nightly mantid-framework --file requirements.txt --file requirements_dev.txt
+   conda activate refred
+
+Then the testsuite can be run via
+
+.. code-block:: bash
+
+   pytest --cov
+
+or ``python path/to/testfile.py`` to run an individual test.
```

Comment:
Not every single test include the lines `if __name__ == '__main__': pytest.main([__file__])` so I would substitute `python` with `pytest`